PRD.md

```diff
@@ -0,0 +1,32 @@
+# Finding National Register Of Historic Places.PRD
+
+## Purpose
+
+This app is built for tourists, architects, planners, real estate developers and tenants. 
+* For tourists and architects, this app will help them find the historic sites on map, read the introductions of the buildings, and add their own notes.
+* For planners and developers, this app can provide information on historic buildings within the land for their reference when they are developing the area.
+* For tenants, this app can help them know better of the house when they are looking for a proud home.
```

Comment:
I would encourage you to focus in on one (_maybe_ two) of these user groups as opposed to building for all of them at once. Having a more narrow description of your user will often allow you to design a more cohesive tool, especially in a short amount of time.

Reply:
Hi Mjumbe,
Thanks for your advice! I think our app's features may work for all three groups, they just have different propensities when using them, so that's why I divided users into three categories to discuss usage scenarios. In subsequent development, we will follow your comment and try to focus more on generic features.

Reply:
Hi Mjumbe @mjumbewu ,
Thanks for your advice! I think our app's features may work for all three groups, they just have different propensities when using them, so that's why I divided users into three categories to discuss usage scenarios. In subsequent development, we will follow your comment and try to focus more on generic features.